### PR TITLE
Ag shoutfactory fix

### PR DESCRIPTION
**Pull request by anuragdce**

### CODE REVIEW

File: ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java [hunk 1]
```diff
@@ -2779,6 +2779,28 @@ private ContentContainer createFavoritesContentContainer(ContentContainer parent
...
+            ContentContainer sliderContainer = createHeroSliderContentContainer(root);
+            root.getContentContainers().add(sliderContainer);
+            contentContainer = sliderContainer;
+        }
+
+        return contentContainer;
+    }
+
+    private ContentContainer createHeroSliderContentContainer(ContentContainer parent) {
+        ContentContainer rootFavorites = new ContentContainer(ZypeSettings.ROOT_SLIDERS_PLAYLIST_ID);
```
Comment (echedev) [inline ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: Can we rename to `rootSliders` to not be confused?

**Review by echedev — APPROVED** [review]

File: ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java (hunk 1 above)
Comment (anuragdce) [reply ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: addressed, thanks

Comment (echedev) [reply ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: oh I'm sorry, I mean to rename the `rootFavorites` to `rootSliders` in the row
```
ContentContainer rootFavorites = new ContentContainer(ZypeSettings.ROOT_SLIDERS_PLAYLIST_ID);
```

Comment (anuragdce) [reply ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: fixed

Comment (echedev) [reply ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: Looks like you changed the `rootFavorites` in `createFavoritesContentContainer()`. Is it intended?
Please check your commit https://github.com/zype/zype-firebuilder/pull/21/commits/d59cfed77c4cfe45a7ade0bf86d7bb7de7d232c8. You added `createHeroSliderContentContainer()` and there was `rootFavorites` variable in it. I only mean to change that variable to make its name consistent.

Comment (anuragdce) [reply ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: addressed, pls check it thanks

**Review by echedev — APPROVED** [review]

**Pull request merged**